### DISCUSSION TITLE
QUERY DATABASE READER

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5764] [Q&A]
Hello! Somebody could help me?
I have a query which in a tool that I use to execute queries, it is executed without problem, which means that it is well done, however in 1 channel that I have a database reader as a connector, I add this query and it fails in the CASE clause, please if someone has happened to share a solution or help, I have tried some things but without success.

![image](https://user-images.githubusercontent.com/98954172/234908716-018440be-6375-4a2f-96ca-55c2860e2351.png)

**Comment by pacmano1** [comment]
You need to post your DB engine type and version.  What jdbc driver you are using, what the exact error is and so on.
